### PR TITLE
Legal documents & small fixes

### PR DESCRIPTION
**Pull request by mikolajmirko**
Changes: look at commit comments
obraz

### CODE REVIEW

File: lib/widgets/dialogs/app_info_dialog.dart [hunk 1]
```diff
@@ -95,16 +77,25 @@ class _AppInfoDialogState extends State<AppInfoDialog> {
 											child: Icon(Icons.keyboard_arrow_right, color: Colors.grey)
 										),
 										leading: Icon(Icons.filter_vintage),
-										onTap: () => _openBrowserpage(context, githubURL)
+										onTap: () => ExternalURL.github.openBrowserPage(context)
```
Comment (stasgora) [inline lib/widgets/dialogs/app_info_dialog.dart:80]: Should _"Project page"_ go to github or website..

File: lib/widgets/dialogs/app_info_dialog.dart [hunk 2]
```diff
@@ -95,16 +77,25 @@ class _AppInfoDialogState extends State<AppInfoDialog> {
...
 									ListTile(
-										title: Text(AppLocales.of(context).translate('$_settingsKey.terms')),
+										title: Text(AppLocales.of(context).translate('$_settingsKey.termsOfUse')),
 										trailing: Padding(
 											padding: EdgeInsets.only(left: 4.0, top: 2.0),
 											child: Icon(Icons.keyboard_arrow_right, color: Colors.grey)
 										),
 										leading: Icon(Icons.description),
-										onTap: () => _openBrowserpage(context, termsURL)
+										onTap: () => ExternalURL.termsOfUse.openBrowserPage(context)
```
Comment (stasgora) [inline lib/widgets/dialogs/app_info_dialog.dart:89]: One not so great thing about our header - it takes all the screen on mobile. You don't even know there is anything further down. We should at least have the page title visible at all times
image

**Review by stasgora — CHANGES_REQUESTED** [review]

File: lib/widgets/forms/plan_form.dart [hunk 3]
```diff
@@ -146,6 +147,10 @@ class _PlanFormState extends State<PlanForm> {
 						onTapped: onChange != null ? () => onChange(item.value, !checked) : null,
 						isActive: checked
 					)
+				),
+				emptyBuilder: (str) => AppHero(
```
[the remark this answers is not in the document]
Comment (mikolajmirko) [reply lib/widgets/forms/plan_form.dart:151]: And I thought I looked everywhere :P Thanks.
Other selects have only app content or are blocked from ever showing up when there is nothing to display.

File: lib/widgets/dialogs/app_info_dialog.dart (hunk 2 above)
Comment (mikolajmirko) [reply lib/widgets/dialogs/app_info_dialog.dart:89]: True, not very user friendly as of now.

Comment (stasgora) [reply lib/widgets/dialogs/app_info_dialog.dart:89]: You could get rid of some stuff for now: there is no pl language, github button is not necessary on the sub-pages..

Comment (mikolajmirko) [reply lib/widgets/dialogs/app_info_dialog.dart:89]: Yep, that's what I have in mind too.

File: lib/pages/caregiver/calendar_page.dart [hunk 4]
```diff
@@ -283,7 +284,11 @@ class _CaregiverCalendarPageState extends State<CaregiverCalendarPage> with Tick
 	          onTapped: onChange != null ? () => onChange(item.value, !checked) : null,
 	          isActive: checked
 	        )
-	      )
+	      ),
+				emptyBuilder: (str) => AppHero(
+					icon: Icons.warning,
+					title: AppLocales.of(context).translate('page.caregiverSection.planForm.fields.assignedChildren.emptyList'),
```
Comment (stasgora) [inline lib/pages/caregiver/calendar_page.dart:290]: That's not going to work very well.. "..you can continue creating a plan.."

Comment (mikolajmirko) [reply lib/pages/caregiver/calendar_page.dart:290]: Frick, I forgot I added this second part to this.

Comment (mikolajmirko) [reply lib/pages/caregiver/calendar_page.dart:290]: Will just the first part of the sentence do the trick?
_"You haven't added any child profile to your account yet"_
I'm creating separate text key anyway, Thinking about writing some additional part like "you can't filter right now" or sth.

Comment (stasgora) [reply lib/pages/caregiver/calendar_page.dart:290]: Great - short & generic. You could add the 2nd part above and format similarly:
image

Comment (mikolajmirko) [reply lib/pages/caregiver/calendar_page.dart:290]: Currently I'm using a general widget `AppHero` that handles just icon, text and bonus action. Do you have any specific title to add?

Comment (stasgora) [reply lib/pages/caregiver/calendar_page.dart:290]: I was thinking about your "you can't filter right now"

Comment (mikolajmirko) [reply lib/pages/caregiver/calendar_page.dart]: obraz

**Review by stasgora — APPROVED** [review]

**Pull request merged**